tests/js/realm-tests.js

```diff
@@ -117,6 +117,16 @@ module.exports = BaseTest.extend({
         TestCase.assertEqual(Realm.defaultPath, newPath, "defaultPath should have been updated");
     },
 
+    testRealmSchemaVersion: function() {
+        TestCase.assertEqual(Realm.schemaVersion(Realm.defaultPath), 0xFFFFFFFFFFFFFFFF);
```

Comment:
This is the version stored internally and is consistent with all the other bindings. Since realms should be compatible across bindings maybe we should leave it? Could also make it a constants.